sdcm/tester.py

```diff
@@ -748,6 +750,28 @@ def get_stress_results_bench(self, queue):
             self.update_stress_results(results)
         return results
 
+    @clean_resources_on_exception
```

Comment:
removed, but why?

Reply:
if you look on `clean_resources_on_exception` implementation it runs tester's `clean_resources`. This is a legacy potentially buggy code (can get  the code to infinite recursion) since there should be only one place to clean resources and it is `tearDown` (it is run by the avocado framework in any case)